Library/Formula/mruby-cli.rb

```diff
@@ -0,0 +1,15 @@
+class MrubyCli < Formula
+  desc "utility for setting up a CLI with mruby"
+  homepage "https://github.com/hone/mruby-cli"
+  url "https://github.com/hone/mruby-cli/releases/download/v0.0.4/mruby-cli-0.0.4-x86_64-apple-darwin14.tgz"
```

Comment:
This needs to be built from source I'm afraid, or submitted to homebrew/binary and/or Caskroom/cask.